README.md

```diff
@@ -1,35 +1,42 @@
-# OpenPower Firmware Build Environment
+# OpenPOWER Firmware Build Environment
 
-The OpenPower firmware build process uses Buildroot to create a toolchain and
+The OpenPOWER firmware build process uses Buildroot to create a toolchain and
 build the various components of the PNOR firmware, including Hostboot, Skiboot,
 OCC, Petitboot etc.
 
 ## Development
 
-Issues, Milestones, pull requests and code hosting is on github:
+Issues, Milestones, pull requests and code hosting is on GitHub:
 https://github.com/open-power/op-build
 
-Mailing list: openpower-firmware@lists.ozlabs.org  
-Info/Subscribe: https://lists.ozlabs.org/listinfo/openpower-firmware  
-Archives: https://lists.ozlabs.org/pipermail/openpower-firmware/  
+Mailing list: openpower-firmware@lists.ozlabs.org
+Info/Subscribe: https://lists.ozlabs.org/listinfo/openpower-firmware
+Archives: https://lists.ozlabs.org/pipermail/openpower-firmware/
 
 ## Building an image
 
+To build an image for a Palmetto system:
+
 ```
 git clone --recursive git@github.com:open-power/op-build.git
 cd op-build
 . op-build-env
 op-build palmetto_defconfig && op-build
 ```
 
-This will build an image for a Palmetto system. There exists default
-configurations for other platforms in openpower/configs/ such as
-Habanero and Firestone.
+There are also default configurations for other platforms in
+`openpower/configs/` such as Habanero and Firestone.
+
+Buildroot/op-build supports both native and cross-compilation - it will
+automatically download and build an appropriate toolchain as part of the build
+process, so you don't need to worry about setting up a
+cross-compiler. Cross-compiling from a x86-64 host is officially supported (it's
+what most developers use).
```

Comment:
I would not recommend this parenthetical. It kind of looks bad to admit that we don't use Power ourselves as much as we would like to.